src/DynamoCore/Library/XmlDocumentationExtensions.cs

```diff
@@ -211,12 +217,20 @@ private static string PrimitiveMap(string s)
                     return "System.Object";
                 case "double":
                     return "System.Double";
+                case "double[]":
+                    return "System.Double[]";
```

Comment:
@mmisol this could also be an IEnumerable or IList, etc., right? Is that why you say this is not foolproof?

Reply:
@aparajit-pratap Exactly